Cargo.toml

```diff
@@ -63,3 +71,10 @@ members = [
 [dev-dependencies]
 uuid = { version = "0.8", features = ["v4", "serde"] }
 difference = "2.0"
+
+# dev-dependencies for json-ld
+#async-std = { version = "^1.5", features = ["attributes"] }
+static-iref = "^1.0"
+iref-enum = "^1.2"
+stderrlog = "^0.5"
+tokio = { version = "^0.2", features = ["macros"] }
```

Comment:
That's quite old, we know it works with stable async i/o, should we put the latest tokio?

Reply:
Updated to use `1.0` (same as `ssi` is using): `cargo test` passes locally.